aten/src/ATen/native/cuda/Distributions.cu

```diff
@@ -724,13 +724,34 @@ Tensor& normal_out_cuda(Tensor& output, double mean, const Tensor& std, Generato
 }
 
 Tensor& normal_out_cuda(Tensor& output, const Tensor& mean, const Tensor& std, Generator* gen) {
+  bool expandable = are_expandable(mean.sizes(), std.sizes());
+  // allow resize if output is an empty tensor
+  if (output.sizes().equals({0})) {
```

Comment:
I forgot to think about backwards compatibility when suggesting this change. It looks like torch.normal (on CPU) allows resize of the output no matter what its size is. The old (THC) torch.normal on CUDA probably also allows the same thing.

However, the current torch.normal (on CUDA) doesn't allow for any resizing at all, and we've had this behavior for 8 months now (and no one has noticed yet...). Given that, I think it's fine if we don't enable the legacy resize behavior of "resize everything".

Reply:
I don't know what "resize everything" means.  But I don't understand how the "same element, non-broadcastable" form works because add_ and mul_ don't support that.